guide/how-it-works/custom-spending-conditions.md

```diff
@@ -0,0 +1,225 @@
+
+## Use case examples
+
+#### Self-custody
+
+Custom spending conditions can be helpful to decrease the risk of losing funds in personal self-custody context. Let's say that you have a wallet that you use regularly on a weekly basis. You could unlock a recovery spending condition automatically if that wallet has not been used for more than six months, because it's likely that you have lost access to your primary key. The recovery condition could contain an additional key held in a different location. Such a setup does not decrease the wallet security, because the second key is locked under normal circumstances of using the wallet. 
+
+In a multi-key context, the recovery condition could require only one instead of two keys to spend funds after 6 months of inactivity. This will help mitigate the risk of loss of funds in case one key holder loses a key.
+
+#### Company treasuries
+
+Custom spending conditions can also be useful in an organizational context. Companies need to make sure that they retain access to funds even if one or more employees lose their keys. Not only that, employees could also be compromised by a third party or try to obstruct the deployment of funds for their own benefit. Another use case is that a company needs to implement company-specific processes for corporate spending. For example, they would want to create a rule that the CFO must sign transactions for treasury management. 
+
+
+## Building blocks
+
+There are currently two main building blocks that can help us construct custom spending conditions: timelocks and veto keys.
+
+#### Timelocks
+
+Timelocks use the passing of time to define additional rules for how funds can be spent from a wallet.
+
+- Unlock additional spending conditions after a certain amount of time has passed.
+- Restrict individual transactions to not be processed immediately but only at a certain point of time in the future (pre-signed transactions).
+
+#### Veto keys
+
+In a multi-key setup it is possible to define a key hierarchy. This means that you can define keys that are required in every transaction. This technique is sometimes also called a “sovereign veto”, as it grants the holder of a certain key the power to veto any transaction, even if the required amount of signatures would have been otherwise reached.
+
+#### Powered by Miniscript
+
+Wallets that support these kinds of configurations typically rely on [Miniscript](/guide/glossary/#miniscript) to implement them. Miniscript is a simplified version of Bitcoin Script which helps to reduce the complexity of Bitcoin Script, the native programming language for Bitcoin. Miniscript is easier to read by developers, and also allows for various build-tools to help ensure that scripts are safe, valid, and efficient.
```

Comment:
What are the alternatives to Miniscript? Just using regular Bitcoin Script? Or are there other ways?